packages/clay-tooltip/src/ClayTooltip.js

```diff
@@ -187,7 +188,48 @@ class ClayTooltip extends ClayComponent {
 	}
 
 	/**
-	 * Handles transionend event.
+	 * Handles touchstart events.
+	 * @memberOf ClayTooltip
+	 * @param {!Event} event
+	 * @private
+	 */
+	_handleTouchStart(event) {
+		this._timer = setTimeout(() => {
+			const content = this._getContent(event.srcElement);
```

Comment:
This, but the last line, is the same code as present in `_handleMouseEnter`. You can extract it to a method `_showTooltip`

Reply:
Done